src/Components/ADT3DSceneBuilder/Internal/Behaviors/Behaviors.tsx

```diff
@@ -268,7 +268,7 @@ const SceneBehaviors: React.FC<Props> = ({
             </PanelFooter>
             <ConfirmDeleteDialog
                 isOpen={isDeleteDialogOpen}
-                onConfirmDeletion={() => {
+                onConfirm={() => {
```

Comment:
should we wrap the callback in a useCallback not to create it in every render?

Reply:
Done.